resources/test-password-logging.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
```

Comment:
Could use some echoes here and there.

Some Steps in between
Password (not expected) vs actual